start-site/src/main/resources/application.yml

```diff
@@ -446,7 +446,7 @@ initializr:
             - compatibilityRange: "[2.2.0.M1,2.4.0-M1)"
               version: 1.4.0
             - compatibilityRange: "[2.4.0-M1,2.5.0-M1)"
-              version: 1.5.1
+              version: 2.0.0
```

Comment:
To be consistent with the [version matrix](https://github.com/okta/okta-spring-boot/blob/master/README.md#spring-boot-version-compatibility), I think this should be changed.

It's a bit of strange mapping, hence why I asked the question earlier but the proper metadata would be:

```yaml
- compatibilityRange: "[2.4.0-M1,2.4.1)"
  version: 1.5.1
- compatibilityRange: "[2.4.1,2.5.0-M1)"
  version: 2.0.0
```

We can't have overlaps or holes in the range so that looks more consistent to me. Thoughts?

Reply:
@snicoll thanks! the one you propose above looks good.